src/shims/unix/fd.rs

```diff
@@ -182,7 +186,10 @@ impl FileDescription for io::Stdout {
         // the host -- there is no good in adding extra buffering
         // here.
         io::stdout().flush().unwrap();
-        ecx.return_written_byte_count_or_error(result, dest)
+        match result {
+            Ok(rw_bytes) => ecx.return_read_write_success(None, None, false, rw_bytes, dest),
```

Comment:
These calls are very hard to read. What does `None, None, false` mean? I shouldn't have to know the argument names to make sense of a function call.